src/App.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import Header from "./components/Header";
+import DataContent from "../src/components/DataContent";
+import API_KEY from "./heplers/keys";
+import './styles/App.css';
+
+function App() {
+    const [dataTrending, setDataTrending] = useState(null);
+    const [dataSearch, setDataSearch] = useState(null);
+    const [itemSearch, setItemSearch] = useState('');
+    const [count, setCount] = useState(5);
+
+    useEffect(() => {
```

Comment:
Please, move fetches to the separate file

Reply:
done